src/rt/sched/cown.h

```diff
@@ -902,7 +898,7 @@ namespace verona::rt
         {
           assert(Scheduler::local()->mutor == nullptr);
           Scheduler::local()->mutor = receiver;
-          receiver->weak_acquire();
+          Cown::acquire(receiver);
```

Comment:
Out of interest why has this changed from weak to strong.

Reply:
We may need to send it an unmute message, so we need to keep it alive.